frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -256,7 +255,7 @@ class BitBox02 extends Component<Props, State> {
                     return true;
                 }
                 if (errorMessage) {
-                    alertUser(errorMessage);
+                    alertUser(errorMessage, undefined, false);
```

Comment:
I'm not happy with passing `false` this way to force the `alertUser` global to NOT render as dialog..

A possible alternative would be to completely get rid of `alertUser` in bitbox02.tsx

Reply:
this or make `alertUser` take one object argument with keys instead: `alertUser({ message, ... })`. the undefined ones can just be skipped then.

Reply:
I guess like this

```diff
diff --git a/frontends/web/src/components/alert/Alert.tsx b/frontends/web/src/components/alert/Alert.tsx
index ebe83952..be1989a3 100644
--- a/frontends/web/src/components/alert/Alert.tsx
+++ b/frontends/web/src/components/alert/Alert.tsx
@@ -28,11 +28,13 @@ import { Button } from '../forms';
  * @param callback callback function called after user confirm
  * @param asDialog option to opt-out of rendinging as dialog
  */
-let alertUser: (
+let alertUser: (args: AlertUserArgs) => void;
+
+type AlertUserArgs = {
     message: string,
     callback?: () => void,
     asDialog?: boolean,
-) => void;
+};
 
 interface State {
     active: boolean;
@@ -61,11 +63,12 @@ class Alert extends Component<WithTranslation, State> {
         });
     }
 
-    private alertUser = (
-        message: string,
-        callback?: () => void,
-        asDialog: boolean = true,
-    ) => {
+    private alertUser = (args: AlertUserArgs) => {
+        const {
+            message,
+            callback,
+            asDialog = true,
+        } = args;
         this.callback = callback;
         this.setState({
             active: true,
```

Now I feel it was a mistake to touch it. I would rewrite it to be a wrapper component that conditionally shows the alert or the children.

```jsx
<WithAlert message=(warningText)>
    {/* rest of the code */}
```

Reply:
I've changed to `(message: string, options?: AlertUserOptions)`, will do another PR with `<WithAlert>` later.

rebased and squashed into https://github.com/digitalbitbox/bitbox-wallet-app/compare/e403bc0eb80ffc99dda6f14efee33c3838750037..c5972f4a4101659d190aad0753518ad62e1224ff